CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Change Log
 
+## [1.0.0rc1](https://github.com/stellargraph/stellargraph/tree/v1.0.0rc1)
+
+[Full Changelog](https://github.com/stellargraph/stellargraph/compare/v0.11.0...v1.0.0rc1)
+
+### Major features and improvements
+
+- Efforts to improve the getting-started process have continued, focusing on the demos:
```

Comment:
possible reword for this to make it more impactful, something like "Enhanced demonstration notebooks and documentation to make the library more accessible to new users". (note that this is the opening line of our 1.0 release features!)

Reply:
Something slightly related - we've got a big focus on demonstration notebooks - do they deserve their own section in our release notes?  (eg when new notebooks come in to demonstrate algorithms, it's not immediately obvious)

Reply:
It's just that a section which calls out new demo notebooks (separate to the features/algs themselves) would make it very easy for users to jump in without having to read the whole detail.

Reply:
Ah, so like a "here's all the relevant demos for this release" section?